tests/utils/test_io_utils.py

```diff
@@ -84,6 +84,8 @@ def test_read_cameras_txt(self) -> None:
         self.assertEqual(K.fx(), 2332.47)
         self.assertEqual(K.px(), 2028)
         self.assertEqual(K.py(), 1520)
+        self.assertEqual(K.k1(), 0.00400066)
```

Comment:
Could you add a small inline that we read in a COLMAP `SIMPLE_RADIAL` model (with 1 radial distortion coefficient), and implicitly converted it to GTSAM's "Cal3Bundler" (with 2 radial distortion coefficients). (The file doesn't contain a k2).